daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -351,9 +351,6 @@ private[lf] object Speedy {
         compiledPackages.pkgInterface.packageLanguageVersion(tmplId.packageId)
       )
 
-    def normValue(templateId: TypeConName, svalue: SValue): V =
-      svalue.toNormalizedValue(tmplId2TxVersion(templateId))
-
```

Comment:
This was wrongly used. Better to deleted it, and force people to compute the version explicitly before calling `toNormalizedValue`. 
Furthermore it induced most of the time unnecessary recomputation of `tmplId2TxVersion(templateId)`